pkg/apis/config/types.go

```diff
@@ -51,11 +51,20 @@ type NodeResourcesAllocatableArgs struct {
 
 	// Resources to be considered when scoring.
 	// Allowed weights start from 1.
-	// An example resource set might includes "cpu" (millicores) and "memory" (bytes)
+	// An example resource set might include "cpu" (millicores) and "memory" (bytes)
 	// with weights of 1<<20 and 1 respectfully. That would mean 1 MiB has equivalent
 	// weight as 1 millicore.
 	Resources []schedulerconfig.ResourceSpec `json:"resources,omitempty"`
 
 	// Whether to prioritize nodes with least or most allocatable resources.
 	Mode *ModeType `json:"mode,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type CapacitySchedulingArgs struct {
+	metav1.TypeMeta
+
+	// KubeConfigPath is the path of kubeconfig.
+	KubeConfigPath *string
```

Comment:
Let's make it (internal version) string.

Reply:
Done